src/main/java/duke/Duke.java

```diff
@@ -17,10 +19,13 @@
     private static final String BACKUP_EXTENSION = ".backup";
     private static final File DEFAULT_USER_DIRECTORY = new File("data" + File.separator + "duke");
     private static final File EXPENSES_FILE = new File(DEFAULT_USER_DIRECTORY, "expenses.txt");
+    private static final File BUDGET_FILE = new File(DEFAULT_USER_DIRECTORY, "budget.txt");
     private static final File EXPENSES_BACKUP_FILE =
-        new File(DEFAULT_USER_DIRECTORY, "expenses.txt" + BACKUP_EXTENSION);
-
-    ExpenseList expenseList;
+            new File(DEFAULT_USER_DIRECTORY, "expenses.txt" + BACKUP_EXTENSION);
+    private static final File BUDGET_BACKUP_FILE =
+            new File(DEFAULT_USER_DIRECTORY, "budget.txt" + BACKUP_EXTENSION);
+    public Budget budget;
+    public ExpenseList expenseList;
```

Comment:
might be best to create a new class to handle loading of files eventually; will add to issue tracker but this is fine for now